src/config.js

```diff
@@ -1049,16 +1095,20 @@ class Config {
         this[settingsToChange] = value
       }
     }
-    return this.emitChangeEvent()
+    return this.emitChangeEvent() && this.emitUpdateEvent(value)
   }
 
   observeKeyPath (keyPath, options, callback) {
     callback(this.get(keyPath))
     return this.onDidChangeKeyPath(keyPath, event => callback(event.newValue))
   }
 
+  listenKeyPath (keyPath, options, callback) {
+    return this.onDidUpdateKeyPath(keyPath, event => callback(event.newValue))
+  }
+
   onDidChangeKeyPath (keyPath, callback) {
-    let oldValue = this.get(keyPath)
+    let oldValue = this.get(keyPath);
```

Comment:
Atom discourages the use of semicolons.
Before:
`let oldValue = this.get(keyPath);`
After:
`let oldValue = this.get(keyPath)`